### PR TITLE
Brackets update 1.14

### PR DESCRIPTION
**Pull request by LodanZark**
Brackets update 1.14

### CODE REVIEW

**Comment by LodanZark** [comment]
builddir/files/share/appdata/io.brackets.Brackets.appdata.xml: FAILED:
• tag-missing           : <translation> not specified
• tag-missing           : <update_contact> is not present
• value-missing         :  is not present
• attribute-invalid     : <screenshot> height too large [https://github.com/adobe/brackets/wiki/screenshots/brackets-ui.png] maximum is 900px
• style-invalid         : <image> has vertical padding [https://github.com/adobe/brackets/wiki/screenshots/brackets-ui.png]
• style-invalid         : <image> has horizontal padding [https://github.com/adobe/brackets/wiki/screenshots/brackets-ui.png]
• aspect-ratio-invalid  : <screenshot> aspect ratio not 16:9 [https://github.com/adobe/brackets/wiki/screenshots/brackets-ui.png]
• style-invalid         : <image> has vertical padding [https://github.com/adobe/brackets/wiki/screenshots/brackets-quick-edit.png]
• style-invalid         : <image> has horizontal padding [https://github.com/adobe/brackets/wiki/screenshots/brackets-quick-edit.png]
• aspect-ratio-invalid  : <screenshot> aspect ratio not 16:9 [https://github.com/adobe/brackets/wiki/screenshots/brackets-quick-edit.png]
• translations-required  : <name> has no translations
• translations-required  : <summary> has no translations
• translations-required  : <description> has no translations

**Comment by paulcarroty** [comment]
@LodanZark hey, are you interested to me a maintainer of this app?

**Comment by LodanZark** [comment]
@paulcarroty I don't mind since I use it as my main web dev tool ;)

**Comment by paulcarroty** [comment]
@LodanZark it will be great if someone will keep things up, 'cause I don't use it and even not subscribed to this repository.

File: io.brackets.Brackets.appdata.xml [hunk 1]
```diff
@@ -4,9 +4,10 @@
   <name>Brackets</name>
   <developer_name>Adobe</developer_name>
   <summary>An open source code editor for the web</summary>
-  <metadata_license>MIT</metadata_license>
```
Comment (TingPing) [inline io.brackets.Brackets.appdata.xml]: Unless the author of this file agrees you can't just relicense things.

**Comment by TingPing** [comment]
@LodanZark 	Invite sent. Feel free to poke me if you have any questions.

**Comment by LodanZark** [comment]
@TingPing  thank you

**Pull request merged**